src/PSRule.Rules.Azure/rules/Azure.APIM.Rule.ps1

```diff
@@ -279,6 +279,14 @@ Rule 'Azure.APIM.MultiRegionGateway' -Ref 'AZR-000341' -Type 'Microsoft.ApiManag
     }
 }
 
+# Synopsis: Wildcard * for any configuration option in CORS policies settings should not be used.
+Rule 'Azure.APIM.CORSPolicy' -Ref 'AZR-000360' -Type 'Microsoft.ApiManagement/service', 'Microsoft.ApiManagement/service/apis', 'Microsoft.ApiManagement/service/policies', 'Microsoft.ApiManagement/service/apis/resolvers', 'Microsoft.ApiManagement/service/apis/operations', 'Microsoft.ApiManagement/service/apis/resolvers/policies', 'Microsoft.ApiManagement/service/products/policies', 'Microsoft.ApiManagement/service/apis/policies',
+'Microsoft.ApiManagement/service/apis/operations/policies' -If { UtilsCORSPolicy -FunctionHelperName 'HasCORSPolicyPattern' } -Tag @{ release = 'GA'; ruleSet = '2023_03'; } {
+    $BadCORSPolicy = UtilsCORSPolicy
+    $Assert.GreaterOrEqual($BadCORSPolicy, '.', 1)
+
```

Comment:
```suggestion
$policies = GetAPIMPolicyNode -Node 'cors';
    foreach ($policy in $policies) {
        Write-Debug "Got policy: $($policy.OuterXml)"
        $allowedHeaders = @($policy.'allowed-headers'.header)
        $Assert.NotIn($allowedHeaders, '.', '*')

    }
```

Reply:
@BernieWhite Isn't it `origin` and `method` we are supposed to check, and not `header`?

![image](https://user-images.githubusercontent.com/99641908/227887287-5abc41f0-3102-4529-a9d9-31f4bc7ad1d1.png)

Reply:
Anything that supports adding a `*`, which is all of the policy elements in your screenshot. `origin` and `header` are single values, the other policy elements are multi-value.

For example:

```powershell
        $allowedHeaders = @($policy.'allowed-headers'.header)
        $Assert.NotIn($allowedHeaders, '.', '*')

        $allowedOrigins = @($policy.'allowed-origins'.origin)
        $Assert.NotIn($allowedOrigins , '.', '*')

        $origin = @($policy.origin)
        $Assert.NotIn($origin, '.', '*')
```